numpy/__init__.pyi

```diff
@@ -5090,7 +5085,7 @@ class matrix(ndarray[_2DShapeT_co, _DType_co]):
     @property
     def T(self) -> matrix[_2D, _DType_co]: ...
     @property
-    def I(self) -> matrix[_2D, Any]: ...
+    def I(self) -> matrix[_2D, Any]: ...  # noqa: E743
```

Comment:
What was the problem here?

Reply:
[E743](https://docs.astral.sh/ruff/rules/ambiguous-function-name/) checks for the use of the characters `l`, `O`, or `I` as function names. We could disable it globally, as `I` makes sense in the context of numerical programming.